jihyeon/app.js

```diff
@@ -32,10 +32,30 @@ app.get('/ping', function (req, res, next){
     return res.status(200).json({message: 'pong'})
 });
 
+app.post('/users/signup', async function (req, res) {
+    const { email, password, name, age, phone_number } = req.body
+    await dataSource.query(`
+        INSERT INTO users (
+            email,
+            password,
+            name,
+            age,
+            phone_number
+        ) VALUES (
+            ?,
+            ?,
+            ?,
+            ?,
+            ?
+        )
+    `,[email, password, name, age, phone_number])
+    res.status(200).json({message: 'userCreated'})
+});
```

Comment:
```suggestion
app.post('/users/signup', async function (req, res) {
    const { email, password, name, age, phone_number } = req.body
    
    await dataSource.query(`
        INSERT INTO users (
            email,
            password,
            name,
            age,
            phone_number
        ) VALUES (
            ?,
            ?,
            ?,
            ?,
            ?
        )
    `,[email, password, name, age, phone_number])
    
    res.status(201).json({message: 'userCreated'})
});
```
1. 코드간 적절한 줄바꿈을 활용해 가독성을 높일 수 있도록 해주세요!
2. `POST`라는 http request method에 맞는 보다 적합한 status code를 반환할 수  있도록 수정되면 좋을 것 같습니다.